doc/nrf/libraries/modem/at_shell.rst

```diff
@@ -0,0 +1,50 @@
+.. _lib_at_shell:
+
+AT shell
+########
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can use this library to add an AT shell command to a shell.
+This enables you to send AT commands to the modem and receive responses.
+When the library is enabled, the shell will have a command ``at``.
+
+You can also enable AT command notifications and disable them.
+
+* Enable AT command notifications:
+
+  .. code-block:: console
+
+     at events_enable
+
+* Disable AT command notifications:
+
+  .. code-block:: console
+
+     at events_disable
+
+Configuration
+*************
+
+The library expects that you have enabled a shell.
+To enable and configure the shell, use the following Kconfig options:
+
+* :kconfig:`CONFIG_AT_SHELL` - Enables the AT Host library
+* :kconfig:`CONFIG_AT_SHELL_CMD_RESPONSE_MAX_LEN` - Maximum AT command response length
+
```

Comment:
Although it should be clear, should we mention that to enable the shell, `CONFIG_SHELL` should be set.

Reply:
Will add a sentence about this